File: STRIVER/video/vid-5/pattern5/p5.cpp
//   * * * *
//   * * * 
//   * *
//   *

#include <iostream>
using namespace std;

int main()
{
    int row = 5;

    for (int i = row; i >= 0; i--)
    {
        for (int j = i; j >= 0; j--)
        {
            cout <<"*"<<" ";
        }
        cout << endl;
    }
}

File: STRIVER/video/vid-5 (Patterns)/pattern5/p5.py
#  * * * *
#  * * * 
#  * *
#  *

def p3(row:int)->None:
    for i in range(row, 0,-1):
        for j in range(i, 0, -1):
            print("*",end=" ")
        print("")

row = 5
p3(row)